src/main/java/org/codeforamerica/shiba/output/documentfieldpreparers/NoSocialSecurityNumberPreparer.java

```diff
@@ -17,18 +17,18 @@
 @Component
 public class NoSocialSecurityNumberPreparer implements DocumentFieldPreparer {
 
-	@Override
-	public List<DocumentField> prepareDocumentFields(Application application, Document document, Recipient recipient) {
-	    PagesData pagesData = application.getApplicationData().getPagesData();
-	    String firstValue = getFirstValue(pagesData, PERSONAL_INFO_NO_SSN);
-	    String ssn = getFirstValue(pagesData, PERSONAL_INFO_SSN);
-	    if (firstValue == null && ssn.isBlank()) {//checkbox not selected and ssn is blank
-	    	return List.of(); //do not check either radio button
-	    }else if(firstValue == null) {
-	    	return List.of(new DocumentField("personalInfo", "noSSNCheck", "true", SINGLE_VALUE));//yes radio button
-	    }else {
-	    	return List.of(new DocumentField("personalInfo", "noSSNCheck", "false", SINGLE_VALUE));//no radio button
-	    }
-	}
+    @Override
+    public List<DocumentField> prepareDocumentFields(Application application, Document document, Recipient recipient) {
+        PagesData pagesData = application.getApplicationData().getPagesData();
+        String noSSNCheckbox = getFirstValue(pagesData, PERSONAL_INFO_NO_SSN);
```

Comment:
Maybe a comment to indicate that this checkbox only appears if it's a certain pops app? Took me a second to figure out what checkbox you were talking about before I went digging into pages-config and saw that condition.